services/users-ms/lambdas/findUser.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-console */
+import to from 'await-to-js';
+
+import config from '../../../config';
+import * as dynamoDb from '../../../libs/dynamoDb';
+import { putEvent } from '../../../libs/awsEventBridge';
+
+export async function main(event) {
+  const userDetail = event.detail;
+  const { personalNumber } = userDetail;
+
+  const [getUserError, userItem] = await to(getUser(personalNumber));
+  if (getUserError) {
+    return console.error('(users-ms: findUser) getUserError', getUserError);
+  }
+
+  if (userItem == undefined) {
+    const [emitEventError] = await to(emitEventUserNotFound(userDetail));
```

Comment:
To keep the code consistent i would suggest to build on the putEvent naming convention, emitEvent is a new naming for the same functionality. This made me a bit confused, due to the knowledge of our putEvent function from the libs.

putEvent in itself might not be the best name for this function but it is what is used currently, so would be good if we could stick to that for know :)